trusted_os/handler.go

```diff
@@ -36,6 +36,7 @@ var irqHandler = make(map[int]func())
 
 // defined in handler.s
 func wakeHandler(g uint32, p uint32)
+func wakeHandlerPreGo122(g uint32, p uint32)
```

Comment:
```suggestion
func wakeHandlerPreGo123(g uint32, p uint32)
```

?? That's the name in the `.s` file (and also the name I would have expected it to have)